<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="utf-8">
    <title>Alex To at SHSU | Projects</title>
    <link rel="stylesheet" href="styles/Dino.css">
</head>
<body>
    <header>
        <img src="images/210811163224-02-china-dinosaurs.jpeg" alt="profile picture" height="250" width="200">
        <h1 class="shadow">How We are All Connected</h1>
        <h2 class="shadow">The world is not what you know...</h2>
    </header>
<article>
<h3>Introduction to Phylogenies</h3>
<br>

<p>Phylogenies are intricate maps of evolutionary relationships, tracing the genetic connections among organisms throughout history. They offer a glimpse into the vast tapestry of life, illuminating the shared ancestry and divergent paths of species across time. At their core, phylogenies are constructed based on the analysis of genetic data, such as DNA sequences, enabling scientists to decipher the evolutionary twists and turns that have shaped the natural world.
These evolutionary trees not only provide insights into the past but also serve as invaluable tools for understanding the present and predicting the future. By mapping the evolutionary history of organisms, phylogenies help scientists unravel the mysteries of biodiversity, uncovering patterns of speciation, adaptation, and extinction. They offer a framework for studying the ecological and evolutionary dynamics of ecosystems, shedding light on how organisms interact and evolve in response to changing environments.
Moreover, phylogenies have practical applications across various fields, from medicine to conservation biology. In medicine, they aid in tracing the origins and transmission pathways of diseases, informing strategies for disease prevention and control. In conservation biology, phylogenetic analyses guide efforts to preserve evolutionary diversity and prioritize species for conservation action.
In essence, phylogenies are windows into the evolutionary past, offering a roadmap for exploring the intricate web of life on Earth and guiding our efforts to conserve and understand it.
</p>
</article>

<article>
    <h3>What is Taxonomy?</h3>
    <p>Taxonomy is the science of naming, describing and classifying organisms and includes all plants, animals and microorganisms of the world. Using morphological, behavioural, genetic and biochemical observations, taxonomists identify, describe and arrange species into classifications, including those that are new to science. Taxonomy identifies and enumerates the components of biological diversity providing basic knowledge underpinning management and implementation of the Convention on Biological Diversity. Unfortunately, taxonomic knowledge is far from complete. In the past 250 years of research, taxonomists have named about 1.78 million species of animals, plants and micro-organisms, yet the total number of species is unknown and probably between 5 and 30 million.
    </p>
    <h4>How to Name a Species: the Taxonomic Process </h4>
     <P>Taxonomists begin by sorting specimens to separate sets they believe represent species. Once the specimens are sorted the next job is to see whether or not they already have names. This may involve working through identification guides, reading descriptions written perhaps 200 years ago, and borrowing named specimens from museums or herbaria to compare with the sample. Such comparison may involve external characters, need to dissect internal structures, or even molecular analysis of the DNA. If there is no match the specimens may represent a new species, not previously given a name. The taxonomist then has to write a description, including ways in which the new species can be distinguished from others, and make up a name for it, in a Latin format. The name and the description must then be properly published so that other taxonomists can see what has been done, and be able to identify the species themselves. From finding the specimens to the name appearing in print can take several years.</P>
    </article>

    <article>
        <h3>Phylogeny vs Taxonomy</h3>
        <br>
        <p>In phylogenetic analysis, taxon sampling selects a small group of taxa to represent the evolutionary history of its broader population.[16] This process is also known as stratified sampling or clade-based sampling.[17] The practice occurs given limited resources to compare and analyze every species within a target population.[16] Based on the representative group selected, the construction and accuracy of phylogenetic trees vary, which impacts derived phylogenetic inferences.[17]

            Unavailable datasets, such as an organism's incomplete DNA and protein amino acid sequences in genomic databases, directly restrict taxonomic sampling.[17] Consequently, a significant source of error within phylogenetic analysis occurs due to inadequate taxon samples. Accuracy may be improved by increasing the number of genetic samples within its monophyletic group. Conversely, increasing sampling from outgroups extraneous to the target stratified population may decrease accuracy. Long branch attraction is an attributed theory for this occurrence, where nonrelated branches are incorrectly classified together, insinuating a shared evolutionary history.[16]
            
            
            Percentage of inter-ordinal branches reconstructed with a constant number of bases and four phylogenetic tree construction models; neighbor-joining (NJ), minimum evolution (ME), unweighted maximum parsimony (MP), and maximum likelihood (ML). Demonstrates phylogenetic analysis with fewer taxa and more genes per taxon matches more often with the replicable consensus tree. The dotted line demonstrates an equal accuracy increase between the two taxon sampling methods. Figure is property of Michael S. Rosenberg and Sudhir Kumar as presented in the journal article Taxon Sampling, Bioinformatics, and Phylogenomics.[17]
            There are debates if increasing the number of taxa sampled improves phylogenetic accuracy more than increasing the number of genes sampled per taxon. Differences in each method's sampling impact the number of nucleotide sites utilized in a sequence alignment, which may contribute to disagreements. For example, phylogenetic trees constructed utilizing a more significant number of total nucleotides are generally more accurate, as supported by phylogenetic trees' bootstrapping replicability from random sampling.
            
            The graphic presented in Taxon Sampling, Bioinformatics, and Phylogenomics, compares the correctness of phylogenetic trees generated using fewer taxa and more sites per taxon on the x-axis to more taxa and fewer sites per taxon on the y-axis. With fewer taxa, more genes are sampled amongst the taxonomic group; in comparison, with more taxa added to the taxonomic sampling group, fewer genes are sampled. Each method has the same total number of nucleotide sites sampled. Furthermore, the dotted line represents a 1:1 accuracy between the two sampling methods. As seen in the graphic, most of the plotted points are located below the dotted line, which indicates gravitation toward increased accuracy when sampling fewer taxa with more sites per taxon. The research performed utilizes four different phylogenetic tree construction models to verify the theory; neighbor-joining (NJ), minimum evolution (ME), unweighted maximum parsimony (MP), and maximum likelihood (ML). In the majority of models, sampling fewer taxon with more sites per taxon demonstrated higher accuracy.
            
            Generally, with the alignment of a relatively equal number of total nucleotide sites, sampling more genes per taxon has higher bootstrapping replicability than sampling more taxa. However, unbalanced datasets within genomic databases make increasing the gene comparison per taxon in uncommonly sampled organisms increasingly difficult.[17]
        </p>
        </article>

        <article>
            <h3>blahhhhhhhhhhhhhhh</h3>
            <br>
            <p>In phylogenetic analysis, taxon sampling selects a small group of taxa to represent the evolutionary history of its broader population.[16] This process is also known as stratified sampling or clade-based sampling.[17] The practice occurs given limited resources to compare and analyze every species within a target population.[16] Based on the representative group selected, the construction and accuracy of phylogenetic trees vary, which impacts derived phylogenetic inferences.[17]
    
                Unavailable datasets, such as an organism's incomplete DNA and protein amino acid sequences in genomic databases, directly restrict taxonomic sampling.[17] Consequently, a significant source of error within phylogenetic analysis occurs due to inadequate taxon samples. Accuracy may be improved by increasing the number of genetic samples within its monophyletic group. Conversely, increasing sampling from outgroups extraneous to the target stratified population may decrease accuracy. Long branch attraction is an attributed theory for this occurrence, where nonrelated branches are incorrectly classified together, insinuating a shared evolutionary history.[16]
                
                
                Percentage of inter-ordinal branches reconstructed with a constant number of bases and four phylogenetic tree construction models; neighbor-joining (NJ), minimum evolution (ME), unweighted maximum parsimony (MP), and maximum likelihood (ML). Demonstrates phylogenetic analysis with fewer taxa and more genes per taxon matches more often with the replicable consensus tree. The dotted line demonstrates an equal accuracy increase between the two taxon sampling methods. Figure is property of Michael S. Rosenberg and Sudhir Kumar as presented in the journal article Taxon Sampling, Bioinformatics, and Phylogenomics.[17]
                There are debates if increasing the number of taxa sampled improves phylogenetic accuracy more than increasing the number of genes sampled per taxon. Differences in each method's sampling impact the number of nucleotide sites utilized in a sequence alignment, which may contribute to disagreements. For example, phylogenetic trees constructed utilizing a more significant number of total nucleotides are generally more accurate, as supported by phylogenetic trees' bootstrapping replicability from random sampling.
                
                The graphic presented in Taxon Sampling, Bioinformatics, and Phylogenomics, compares the correctness of phylogenetic trees generated using fewer taxa and more sites per taxon on the x-axis to more taxa and fewer sites per taxon on the y-axis. With fewer taxa, more genes are sampled amongst the taxonomic group; in comparison, with more taxa added to the taxonomic sampling group, fewer genes are sampled. Each method has the same total number of nucleotide sites sampled. Furthermore, the dotted line represents a 1:1 accuracy between the two sampling methods. As seen in the graphic, most of the plotted points are located below the dotted line, which indicates gravitation toward increased accuracy when sampling fewer taxa with more sites per taxon. The research performed utilizes four different phylogenetic tree construction models to verify the theory; neighbor-joining (NJ), minimum evolution (ME), unweighted maximum parsimony (MP), and maximum likelihood (ML). In the majority of models, sampling fewer taxon with more sites per taxon demonstrated higher accuracy.
                
                Generally, with the alignment of a relatively equal number of total nucleotide sites, sampling more genes per taxon has higher bootstrapping replicability than sampling more taxa. However, unbalanced datasets within genomic databases make increasing the gene comparison per taxon in uncommonly sampled organisms increasingly difficult.[17]
            </p>
            </article>

<aside class= "right">
    <h4>References</h4>
    <ol>
        <li><a href="https://evolution.berkeley.edu/evolution-101/the-history-of-life-looking-at-the-patterns/understanding-phylogenies/">Evolution.Berkeley</a></li>
        <li><a href="https://www.britannica.com/science/phylogeny/Taxonomic-systems">Britannica</a></li>
        <li><a href="https://www.nature.com/articles/s41467-022-31511-0">Nature</a></li>
    </ol>
    <a href="https://Bostonsato.github.io">Return to the homepage</a>
</aside>
<footer> 
    <p class="copyright">&copy; 2024 Alex To</p>
</footer>
    <p class="fixed">Contact me at aqt005@shsu.edu</p>
</body>
</html>



